palette/src/hwb.rs

```diff
@@ -340,11 +340,34 @@ where
 {
     type Scalar = T;
 
-    fn lighten(&self, amount: T) -> Hwb<S, T> {
+    fn lighten(&self, factor: T) -> Hwb<S, T> {
+        let difference_whiteness = if factor >= T::zero() {
+            T::max_intensity() - self.whiteness
+        } else {
+            self.whiteness
+        };
+        let delta_whiteness = difference_whiteness.max(T::zero()) * factor;
+
+        let difference_blackness = if factor >= T::zero() {
+            T::max_intensity() - self.blackness
+        } else {
+            self.blackness
+        };
+        let delta_blackness = difference_blackness.max(T::zero()) * factor;
```

Comment:
This would need to be backwards, right? Blackness should decrease when whiteness increases.

Reply:
Right, and the sign should be switched when creating the color too I think.

Reply:
Yes, exactly. That should work.